instrumentation/runtime-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetrics/RuntimeMetricsInstaller.java

```diff
@@ -38,6 +39,7 @@ public void afterAgent(Config config, AutoConfiguredOpenTelemetrySdk unused) {
       if (config.getBoolean(
           "otel.instrumentation.runtime-metrics.experimental-metrics.enabled", false)) {
         GarbageCollector.registerObservers(GlobalOpenTelemetry.get());
+        BufferPools.registerObservers(GlobalOpenTelemetry.get());
```

Comment:
Our process for adding these has been to discuss in the metrics JVM SIG, and codify in the [runtime semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics). 

Are you able to attend the [JVM metrics SIG](https://github.com/open-telemetry/community/blame/main/README.md#L128)?

If not, we should invite the participants to have a discussion here.

Reply:
sorry, due to the time difference, i may not be able to attend the SIG meeting.

Reply:
i've requested JVM metrics folks to have a look at this PR

(also @cuichenli in case you didn't notice, the JVM metrics meetings are at alternating times to accommodate different timezones)